### PR TITLE
Fix setup open for py2

### PR DESCRIPTION
**Pull request by mcdonnnj**
# 

## 🗣 Description
This PR fixes an issue where after updating upstream changes the `readme()` function in `setup.py` used a Python 3 only form. This adds in a Python 2 compatible fallback since this script sees primary use in a Python 2 environment at present.

The `py-dateutil` package was also version bounded because of an issue with botocore as seen in https://github.com/boto/botocore/issues/1872

## 💭 Motivation and Context
The script failed to install requirements because of Python 3 only code used in the `readme()` function which hindered successful AMI building.

## 🧪 Testing
I test that `pip install -r requirements.txt` worked correctly in both Python 2 (2.7.16) and Python 3 (3.8.0) environments.

## 🚥 Types of Changes

- [x] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (causes existing functionality to change)

## ✅ Checklist

- [x] My code follows the code style of this project.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [x] I have read the **CONTRIBUTING** document.
- [ ] I have added tests to cover my changes.
- [x] All new and existing tests passed.

### CODE REVIEW

File: setup.py [hunk 1]
```diff
@@ -67,7 +74,7 @@ def package_vars(version_file):
         "docopt >= 0.6.2",
         "mongo-db-from-config @ https://github.com/cisagov/mongo-db-from-config/tarball/develop#egg=mongo-db-from-config",
         "netaddr >= 0.7.10",
-        "python-dateutil >= 2.2",
+        "python-dateutil >= 2.2, < 2.8.1",
```
Comment (dav3r) [inline setup.py:80]: Can you please add a comment in here about why we want to stay below 2.8.1?

**Review by dav3r — APPROVED** [review]
Approved, with one comment requested
🍪

File: setup.py (hunk 1 above)
Comment (mcdonnnj) [reply setup.py:80]: Ah, good point. This should be resolved in 76551ed

**Review by jsf9k — APPROVED** [review]
Nice work!

**Review by felddy — APPROVED** [review]
Nice!  💪

**Pull request merged**